### ISSUE TITLE
[Tested on 1.20.1 and 1.19.2] Forge Item Layers emissivity does not show up properly

### ISSUE THREAD

**Issue by Zepalesque** [FiniteReality/embeddium#32]
While using Embeddium in my development environment for developing my mod, I noticed a strange behavior that I ended up figuring out only happened with embeddium, the emissive item layers would not glow. The blocks work fine, it's just the items that don't.
I tested this on both 1.19.2 and 1.20.1 and it happened on both versions when Embeddium was installed.

**1.19.2, no Embeddium (works properly):**
---
![1 19 2-functional](https://github.com/embeddedt/embeddium/assets/60141811/c39c4a05-8dc9-447c-93ab-787b6c676268)

**1.19.2, with Embeddium (does not work properly):**
---
![1 19 2-embeddium](https://github.com/embeddedt/embeddium/assets/60141811/94fcb221-194d-4058-a9b7-6cd34cb2aa85)

**1.20.1, no Embeddium (works properly):**
---
![1 20 1-functional](https://github.com/embeddedt/embeddium/assets/60141811/bed7f8d2-d1f7-429d-90f6-c0246360fc93)

**1.20.1, with Embeddium (does not work properly):**
---
![1 20 1-embeddium](https://github.com/embeddedt/embeddium/assets/60141811/261026b4-9d53-4537-827e-e2551bfe97b8)

Just in case it is helpful, here are the item json files from the two versions:

1.19.2:
```
{
  "parent": "minecraft:item/generated",
  "emissive_layers": [1],
  "loader": "forge:item_layers",
  "render_types": {},
  "textures": {
    "layer0": "aether_redux:block/natural/glowsprouts",
    "layer1": "aether_redux:block/natural/glowsprouts_glow"
  }
}
```

1.20.1:
```
{
  "parent": "minecraft:item/generated",
  "forge_data": {
    "layers": {
      "1": {
        "block_light": 15,
        "sky_light": 15
      }
    }
  },
  "loader": "forge:item_layers",
  "render_types": {},
  "textures": {
    "layer0": "aether_redux:block/natural/glowsprouts",
    "layer1": "aether_redux:block/natural/glowsprouts_glow"
  }
}
```

Here's the mod download as well just in case you want a specific example to test (note that when this issue is first submitted, the 1.19.2 and 1.20.1 versions will not have been uploaded yet, but hopefully within an hour or so they will):
https://modrinth.com/mod/the-aether-redux

**Comment by embeddedt** [comment]
Thanks for catching this - the block renderer needed a patch to handle Forge emissivity (since that doesn't exist in vanilla); the item renderer probably just needs a similar tweak.

**Comment by embeddedt** [comment]
Fixed in 0.1.7 (only had time to check 1.20.1, but I applied the appropriate fixes on all versions).
